fastlane/actions/build_cleanup.rb

```diff
@@ -0,0 +1,42 @@
+module Fastlane
+  module Actions
+    class BuildCleanupAction < Action
+
+      # Support
+      def self.is_supported?(platform)
+        true
+      end
+
+      # Run
+      def self.run(params)
+        keychain_path = lane_context[:keychain_path] || UI.user_error!("No build keychain requiring cleanup found. Did you run build_setup before build_cleanup?")
+
+        provisioning_profile_destination = "#{Dir.home}/Library/MobileDevice/Provisioning Profiles"
+        provisioning_profile_names = lane_context[:provisioning_profile_names] || UI.user_error!("No provisioning profiles requiring cleanup found. Did you run build_setup before build_cleanup?")
+
+        Fastlane::Actions::DeleteKeychainAction.run(keychain_path: keychain_path)
+
+        provisioning_profile_names.each do |profile_name|
+          profile_path = File.join(provisioning_profile_destination, profile_name)
+          File.delete(profile_path) if File.exist?(profile_path)
+        end
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Performs cleanup after the build, deleting the keychain and removing provisioning profiles"
+      end
+
+      def self.authors
+        ["Pranjal Satija - @pranjalsatija on GitHub"]
```

Comment:
Should `self.authors` be removed from the files? I'm not sure how much value it's adding.

Reply:
I think it can be helpful to quickly identify who is likely most familiar with that particular action, but when that person is no longer easily available, I agree it adds less value.

I'll wait a bit to see if anyone else has any opinions here.

Reply:
I believe the `self.description`, `self.authors`, and `self.available_options` are either automatically generated or standard fields that are expected to be included with plugin documentation. I would advocate for keeping it rather than removing it.

Since the original author might not be as available now as he was previously, maybe we could add another author in this field since it's an array?
```ruby
def self.authors
         ["Pranjal Satija - @pranjalsatija on GitHub",
          "Will McGinty - @wmcginty on GitHub]
end
```

Reply:
Added myself as an author, for now at least. If needed we can discuss a more long term solution.